sdk/textanalytics/Azure.AI.TextAnalytics/src/AbstractSummaryAction.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// A set of options used to configure abstractive summarization, including the model version to use, the maximum
+    /// number of sentences that the resulting summary can have, and more.
+    /// </summary>
+    public class AbstractSummaryAction
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AbstractSummaryAction"/> class.
+        /// </summary>
+        public AbstractSummaryAction()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AbstractSummaryAction"/> class based on the given
+        /// <see cref="AbstractSummaryOptions"/>. It sets the <see cref="ModelVersion"/>,
+        /// <see cref="DisableServiceLogs"/>, and <see cref="MaxSentenceCount"/> properties.
+        /// </summary>
+        public AbstractSummaryAction(AbstractSummaryOptions options)
```

Comment:
.NET Question: In all other action inputs, the `ActionName` is not in the constructor. Do you know the reason?

Reply:
I think the preferred constructor in these *Action classes are the default constructors, and these other constructors that receive an *Options class were added for convenience. However, the *Options classes do not have an `ActionName` property to copy the value from, so that property is not set as part of them.